resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -364,13 +360,5 @@ You can also specify `volcano` tag to only run Volcano test:
 
 ## Cleanup Volcano
 
-    # x86_64
-    kubectl delete -f https://raw.githubusercontent.com/volcano-sh/volcano/v1.5.1/installer/volcano-development.yaml
-
-    # arm64:
-    kubectl delete -f https://raw.githubusercontent.com/volcano-sh/volcano/v1.5.1/installer/volcano-development-arm64.yaml
-    
-    # Cleanup Volcano webhook 
```

Comment:
just double confirm: the cleanup webhook is no longer needed?

Reply:
It's included by the original `YAML` now. So, if we try to clean up here again, it shows error messages because there is no such resource here.